### PR TITLE
Check if the progress panel exists when TEST_FAILED_EVENT fires (close #180)

### LINKED ISSUE #180: We should check if the progress panel exists when TEST_FAILED_EVENT fires
E.g. for this test:
```js
'@test'['Test'] = {
    '1.Step without action': function () {
        this.temp = true;
    },
    '2.Failed step':         function () {
        throw 'ERROR';
    }
};
```

### PR DESCRIPTION
**Pull request by helen-dikareva**
/cc @inikulin @AlexanderMoskovkin

### CODE REVIEW

**Comment by inikulin** [comment]
lgtm

File: src/client/test-run/index.js.mustache [hunk 1]
```diff
@@ -101,11 +101,13 @@
...
 
         runner.on(runner.ACTION_RUN_EVENT, function () {
-            progressPanel.close(true);
+            if(progressPanel)
+                progressPanel.close(true);
         });
 
         runner.on(runner.TEST_FAILED_EVENT, function () {
-            progressPanel.close(false);
+            if(progressPanel)
```
Comment (AlexanderMoskovkin) [inline src/client/test-run/index.js.mustache]: Please, add the space after the `if` here and above, it's our code rule
`if (!progressPanel)`

**Comment by inikulin** [comment]
I'm missing linting so much (

**Comment by helen-dikareva** [comment]
Sorry, it's done. FPR

**Comment by inikulin** [comment]
lgtm

**Comment by AlexanderMoskovkin** [comment]
lgtm

**Pull request merged**